woeip/settings.py

```diff
@@ -40,6 +41,15 @@
 
 INSTALLED_APPS = DJANGO_APPS + THIRD_PARTY_APPS + LOCAL_APPS
 
+
+if len(env('AWS_S3_ACCESS_KEY_ID')) > 10:
```

Comment:
Is this to disable use of S3? If so, this works, but is not very explicit. Consider the alternative:

1. Allow DEFAULT_FILE_STORAGE to be passed as an environment variable, but default to S3. 
2. Set this value to local file storage for local usage in Docker-Compose.local. 
3. Default the S3 settings to None. 
4. If the file storage is set to S3 and the access variables are not set, raise a configuration error.